src/cameras/CubeCamera.js

```diff
@@ -13,12 +11,22 @@ import { PerspectiveCamera } from './PerspectiveCamera.js';
 
 var fov = 90, aspect = 1;
 
-function CubeCamera( near, far, cubeResolution, options ) {
+function CubeCamera( near, far, renderTarget ) {
 
 	Object3D.call( this );
 
 	this.type = 'CubeCamera';
 
+	if ( renderTarget.isWebGLCubeRenderTarget !== true ) {
+
+		console.error( 'THREE.CubeCamera: The constructor now expects an instance of WebGLCubeRenderTarget as third parameter.' );
+		return;
+
+	}
+
+	this.renderTarget = renderTarget;
+	this.renderTarget.texture.name = "CubeCameraTexture";
```

Comment:
I am not a fan of modifying the user's data. So in spite of the fact that this reasonable here, I would restore this line to where it came from. Let the creator of the render target set the name.

Reply:
Let's see... I like adding `.type` to materials because it improves `renderer.info`.

Do we need `texture.name` for anything? It looks like I added it, but I may have copied it from somewhere...

Reply:
> Do we need texture.name for anything?

AFAIK, it has a pure informative purpose.

Reply:
Remove it, I guess. :-)